### ISSUE TITLE
2.3.14 fails to update global schedule

### ISSUE THREAD

**Issue by ghost** [bluecherrydvr/bluecherry-apps-issues#72]
"Could not apply changes at this time, please try later" coming from www/lib/lib.php:134
Reported by Kenneth Engstrom, reproduced by me.

**Comment by ghost** [comment]
Fixed

```
commit 13df6ceda79cf185daded28a5e2cbce3ba61c59a
Date:   Sat Jan 10 23:10:45 2015 +0200

commit 50d0953b4b6036a0c8e5f2e31bd33542c07f144d
Date:   Sat Jan 10 23:04:30 2015 +0200
```

**Comment by ghost** [comment]
The fix is released in 2.3.15.
